airflow/include/exampleplugin.go

```diff
@@ -5,42 +5,65 @@ import "strings"
 // ExamplePlugin created with astro airflow init
 var ExamplePlugin = strings.TrimSpace(`
 from airflow.plugins_manager import AirflowPlugin
-from flask_admin.base import MenuLink
+from airflow.version import version
 
 """
 Look for the Astronomer tab in the UI.
 """
-airflow_plugins_ml = MenuLink(
-    category='Astronomer',
-    name='Airflow-Plugins',
-    url='https://github.com/airflow-plugins/')
-
-astro_docs_ml = MenuLink(
-    category='Astronomer',
-    name='Astronomer Docs',
-    url='https://www.astronomer.io/docs/')
-
-astro_guides_ml = MenuLink(
-    category='Astronomer',
-    name='Airflow Guides',
-    url='https://www.astronomer.io/guides/')
-
-
-class AstroLinksPlugin(AirflowPlugin):
-    name = 'astronomer_menu_links'
-    operators = []
-    flask_blueprints = []
-    hooks = []
-    executors = []
-    macros = []
-    admin_views = []
-    menu_links = [airflow_plugins_ml, astro_docs_ml, astro_guides_ml]
-    appbuilder_views = []
-    appbuilder_menu_items = [
-        {
-            "name": ml.name,
-            "category": ml.category,
-            "category_icon": "fa-rocket",
-            "href": ml.url,
-        } for ml in menu_links
-    ]`)
+airflow_plugins_ml = {
+    "name": "Airflow-Plugins",
+    "category": "Astronomer",
+    "category_icon": "fa-rocket",
+    "href": "https://github.com/airflow-plugins/"
+}
+
+astro_docs_ml = {
+    "name": "Astronomer Docs",
+    "category": "Astronomer",
+    "category_icon": "fa-rocket",
+    "href": "https://www.astronomer.io/docs/"
+}
+
+astro_guides_ml = {
+    "name": "Airflow Guide",
+    "category": "Astronomer",
+    "category_icon": "fa-rocket",
+    "href": "https://www.astronomer.io/guides/"
+}
+
+_appbuilder_menu_items = [airflow_plugins_ml, astro_docs_ml, astro_guides_ml]
+
+# Airflow >= 2.0
+if version.startswith("2"):
```

Comment:
I generally prefer duck-typing to explicit version checks. I wonder if there is something we could do here...

Reply:
Unfortunately, airflow still consumes menu_link and does nothing with it, I will raise a PR to remove that but until than the only other duck-typing that is possible is to try importing `flask-admin`:

```
try:
     from flask_admin.base import MenuLink
     # Airflow <2
except ImportError:
     # Airflow >= 2
```

but that doesn't feel ideal

Reply:
Is there a problem setting `menu_link` if we also set `appbuilder_menu_items`?

Reply:
`menu_link` is passed instance of `MenuLink` from `flask_admin.base` which does not exist for 2.0

Reply:
But to answer your question, there is no problem in setting `menu_link` if we also set `appbuilder_menu_items`?

Reply:
TBH we don't even need to support non-RBAC - we've used RBAC only for a long time, and this is nothing to do with Airflow 2 or not, right?

So we can just only set the FAB-related fields, right?

Reply:
Updated